### PR TITLE
upgrade bn.js and elliptic, add 2 curves

### PR DESCRIPTION
**Pull request by calvinmetcalf**

### CODE REVIEW

**Comment by calvinmetcalf** [comment]
part of crypto-browserify/crypto-browserify#140

File: browser.js [hunk 1]
```diff
@@ -25,12 +25,22 @@ var aliases = {
...
+		name: 'p521',
+		byteLength: 66
 	}
 };
 
 aliases.p224 = aliases.secp224r1;
 aliases.p256 = aliases.secp256r1 = aliases.prime256v1;
 aliases.p192 = aliases.secp192r1 = aliases.prime192v1;
+aliases.p384 = aliases.secp384r1;
+aliases.p521 = aliases.secp521r1;
```
Comment (dcousens) [inline browser.js:43]: ~~is it `521` or `512`?~~

**Comment by dcousens** [comment]
ACK

File: package.json [hunk 2]
```diff
@@ -24,8 +24,8 @@
   },
   "homepage": "https://github.com/crypto-browserify/createECDH",
   "dependencies": {
-    "bn.js": "^2.0.0",
-    "elliptic": "^3.0.0"
+    "bn.js": "^3.0.0",
```
Comment (dcousens) [inline package.json:27]: Not `3.2.0`?

**Comment by calvinmetcalf** [comment]
I'll publish

**Comment by dcousens** [comment]
Check nit above

**Pull request merged**